src/components/welcome/Buttons.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Button } from "@mui/material";
+import styled from "styled-components";
```

Comment:
hmmmmm, yeah, this styled components are real nice, yet I guess that for sake of consistency we should stick with Material UI. Yet, if needed you could customize using Material UI features, please have a look at [styled](https://mui.com/material-ui/customization/how-to-customize/), which allow you to modify the styling whitout the expense of using another library